content/docs/authorization/deployment/powerflex/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: PowerFlex
+linktitle: PowerFlex
+description: >
+  Enabling CSM Authorization for PowerFlex CSI Driver
+---
+
+## Configuring PowerFlex CSI Driver with CSM for Authorization
+
+Given a setup where Kubernetes, a storage system, and the CSM for Authorization Proxy Server are deployed, follow the steps below to configure the CSI Drivers to work with the Authorization sidecar:
```

Comment:
Change "the steps below" to "these steps"

Reply:
fixed.